src/Chainweb/Chainweb/MinerResources.hs

```diff
@@ -86,14 +87,15 @@ withMiningCoordination logger enabled cutDb inner
             { _coordLogger = logger
             , _coordCutDb = cutDb
             , _coordState = t
-            , _coordLimit = 2500
+            , _coordLimit = 1200
```

Comment:
We throttle with 503 if we have more than 1200 pending work items.

Should we make this configurable?

Reply:
Yes.